tests/TestCase/ORM/AssociationTest.php

```diff
@@ -142,6 +142,31 @@ public function testClassNameUnnormalized()
     }
 
     /**
+     * Tests that an exception is thrown when invalid target table is fetched
+     * from a registry.
+     *
+     * @return void
+     * @expectedException \RuntimeException
+     */
+    public function testInvalidTableFetchedFromRegistry()
+    {
+        TableRegistry::get('Test');
+
+        $config = [
+            'className' => '\Cake\Test\TestCase\ORM\TestTable',
+        ];
+        $this->association = $this->getMockBuilder('\Cake\ORM\Association')
```

Comment:
@dereuromark I'm just following a "convention" - all tests do that. We should IMO switch to `Class::class` someday anyway.